.github/workflows/security-build.yml

```diff
@@ -0,0 +1,37 @@
+
+name: Security Build
+on:
+  push:
+  workflow_dispatch: {}
+
+
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
+      - uses: actions/checkout@v3
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          version: latest
```

Comment:
Do we need linting check here, since we already have it run as a "standalone" GH action:
https://github.com/0xPolygon/cdk-data-availability/blob/main/.github/workflows/lint.yml.

The linter check is failing (https://github.com/0xPolygon/cdk-data-availability/actions/runs/8113958647/job/22178495028?pr=61), and it seems to me like checkout code part is lacking here. Can you take a look into it, in case we need linting check here as well?

Reply:
security-build file has been updated, removed as it was already covered, thanks!